bin/install.sh

```diff
@@ -92,7 +94,7 @@ if [ "${OS}" = "mac" ]; then
     hdiutil detach jlmnt
     rm -rf jlmnt julia.dmg
 elif [ "${OS}" = "winnt" ]; then
-    # TODO: I have no idea
+    stop "don't know what to do"
```

Comment:
Is `stop` a POSIX `sh` thing? For some reason I thought it was `bash` but I'm not certain and Google is failing me.